playlist.go

```diff
@@ -190,6 +190,86 @@ func (c *Client) GetPlaylistTracks(
 	return &result, err
 }
 
+// PlaylistItem contains info about an item in a playlist.
+type PlaylistItem struct {
+	// The date and time the track was added to the playlist.
+	// You can use the TimestampLayout constant to convert
+	// this field to a time.Time value.
+	// Warning: very old playlists may not populate this value.
+	AddedAt string `json:"added_at"`
+	// The Spotify user who added the track to the playlist.
+	// Warning: vary old playlists may not populate this value.
+	AddedBy User `json:"added_by"`
+	// Whether this track is a local file or not.
+	IsLocal bool `json:"is_local"`
+	// Information about the track.
+	Track PlaylistItemTrack `json:"track"`
+}
+
+// PlaylistItemTrack is a union type for both tracks and episodes. It defaults to track type.
+type PlaylistItemTrack struct {
+	*FullTrack
```

Comment:
I think it would be nicer if we just made this `Track *FullTrack`

Reply:
Changed it to full field.